### ISSUE TITLE
Option for ignoring minor errors

### ISSUE THREAD

**Issue by mristuccia** [mjeanroy/exiftool#68]
Hello,

is there a way to tell exiftool to ignore minor errors?
The command line parameter is "-m".

Thank you in advance.

**Comment by mjeanroy** [comment]
@mristuccia Not for now, but I don't think it should be hard to add. If you have time to submit a PR, that would be awesome and I will be happy to review it!

**Comment by mjeanroy** [comment]
Hi,
I just published version 2.6.0 to maven central. You can now set `-m` option, for example:

```java
exiftool.getImageMeta(image, StandardOptions.builder()
    .ignoreMinorErrors(true)
    .build());
```
